site/content/docs/Features/_index.md

```diff
@@ -6,7 +6,7 @@ weight: 30
 
 ### Snapshots
 
-Kopia uploads directories and files to remote storage called [Repository](../architecture/) and maintains a set of historical point-in-time snapshot records based on defined policies.
+Kopia uploads directories and files to remote storage called [Repository](https://kopia.io/docs/repositories/) and maintains a set of historical point-in-time snapshot records based on defined policies.
```

Comment:
thanks, but can you do relative link `../repositories/` instead?

Reply:
```suggestion
Kopia uploads directories and files to remote storage called [Repository](../repositories/) and maintains a set of historical point-in-time snapshot records based on defined policies.
```